extensions/container-image/container-image-s2i/deployment/src/main/java/io/quarkus/container/image/s2i/deployment/S2iConfig.java

```diff
@@ -11,7 +11,7 @@
 @ConfigRoot(phase = ConfigPhase.BUILD_TIME)
 public class S2iConfig {
 
-    public static final String DEFAULT_BASE_JVM_IMAGE = "fabric8/s2i-java:2.3";
+    public static final String DEFAULT_BASE_JVM_IMAGE = "registry.access.redhat.com/openjdk/openjdk-11-rhel7";
```

Comment:
I'm not sure if this is a good idea, purely from "legal" point of view. This is RHEL, after all. Too bad UBI Java isn't ready yet :-(

Reply:
@maxandersen any idea about this?